ur_robot_driver/config/ur10_controllers.yaml

```diff
@@ -130,6 +130,36 @@ joint_group_vel_controller:
    type: velocity_controllers/JointGroupVelocityController
    joints: *robot_joints
 
+forward_joint_traj_controller:
+    type: "joint_trajectory_controllers/PassThroughController"
+    joints: *robot_joints
+
+forward_cartesian_traj_controller:
+    type: "cartesian_trajectory_controllers/PassThroughController"
+    joints: *robot_joints
+
+twist_controller:
+  type: "cartesian_ros_controllers/TwistController"
+  frame_id: "tool0_controller"
+  publish_rate: *loop_hz
+  joints: *robot_joints
+
+pose_traj_controller:
+    type: "pose_controllers/CartesianTrajectoryController"
+
+    # UR driver convention
```

Comment:
Could you explain what's the *convention* here that's referenced?

Reply:
[Here's](https://github.com/fzi-forschungszentrum-informatik/cartesian_ros_control/tree/master/cartesian_trajectory_controller#controller-yaml) more documentation on this convention.